startupscript/git-setup.sh

```diff
@@ -46,5 +46,14 @@ ${RUN_AS_LOGIN_USER} "mkdir -p '${WORKBENCH_GIT_REPOS_DIR}'"
 # to the git references, the corresponding git repo cloning will be skipped.
 # Keep this as last thing in script. There will be integration test for git cloning (PF-1660). If this is last thing, then
 # integration test will ensure that everything in script worked.
-${RUN_AS_LOGIN_USER} "cd '${WORKBENCH_GIT_REPOS_DIR}' && wb git clone --all"
-
+pushd "${WORKBENCH_GIT_REPOS_DIR}" || exit 1
+${RUN_AS_LOGIN_USER} "wb resource list --type=GIT_REPO --format json" | \
+  jq -c .[] | \
+  while read -r ITEM; do
+    GIT_REPO_NAME="$(echo "$ITEM" | jq -r .id)"
+    GIT_REPO_URL="$(echo "$ITEM" | jq -r .gitRepoUrl)"
+    if [[ ! -d "${GIT_REPO_NAME}" ]]; then
```

Comment:
My read of the CLI code is that it will continue if one clone fails: https://github.com/verily-src/terra-tool-cli/blob/main/src/main/java/bio/terra/cli/command/app/passthrough/Git.java#L100

I will make this loop follow the same convention and add documentation.

Reply:
Tested that this WAI (continues when a single clone fails).